testing/py3-catkin-pkg-modules/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: Russ Webber <russ@rw.id.au>
+# Maintainer: Russ Webber <russ@rw.id.au>
+pkgname=py3-catkin-pkg-modules
+_pkgname=catkin_pkg_modules
+pkgver=0.4.12
+pkgrel=0
+pkgdesc="Standalone Python library for the catkin build system."
+url="http://wiki.ros.org/catkin_pkg"
+arch="noarch"
+license="BSD-3-Clause"
+depends="py3-docutils py3-dateutil py3-parsing"
+makedepends="python3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	python3 setup.py test
```

Comment:
It does not run any tests. In most cases it is because the pypi pkg does not contain the test suite and you have to use the GH src pkg instead - https://github.com/ros-infrastructure/catkin_pkg/archive/0.4.12.tar.gz

Reply:
Tests are included in the pypi package. Will have a look at why CI is saying 0 tests ran.

Reply:
You are right. Some tests are included but not the data folder.

Upon further research this is required to run tests:
```
$ pip install nose coverage argparse python-dateutil docutils pyparsing
$ pip install flake8 flake8-blind-except flake8-builtins flake8-class-newline flake8-comprehensions flake8-deprecated flake8-docstrings flake8-import-order flake8-quotes
$ nosetests -s --tests test
```
Is this sort of elaborate setup for a test case usually done in APKBUILDs?

Reply:
py package madness... sometimes you only need some of the mentioned depends to pass the tests.

If a few are missing then it is normal to package them or simply add options="!check" # depends not packaged || mentioning the py pkg names.

Reply:
@TBK all the flake8 deps weren't packaged and have made a note indicating this.